ddtrace/contrib/botocore/patch.py

```diff
@@ -640,6 +648,18 @@ def patched_api_call(original_func, instance, args, kwargs):
                                 # The message originated from SQS
                                 message_body = message
                                 context_json = json.loads(message_body["MessageAttributes"]["_datadog"]["StringValue"])
+                            elif (
+                                "MessageAttributes" in message
+                                and "_datadog" in message["MessageAttributes"]
+                                and "BinaryValue" in message["MessageAttributes"]["_datadog"]
+                            ):
+                                # Raw message delivery
+                                message_body = message
+                                context_json = json.loads(
+                                    message_body["MessageAttributes"]["_datadog"]["BinaryValue"].decode()
+                                )
+                            else:
+                                log.debug("DataStreams did not handle message: %r", message)
```

Comment:
Should we make this a bit more specific, like explain that it wasn't handled because it's in a format we don't support? Or is general the best we can do since we don't know why the message wasn't handled?

Reply:
The latter - so far its been an exploration into AWS behavior.  The other cases are the known issues, this catches the unknown unknowns for the purposes of gathering info from customers.